tidb-cloud/serverless-faqs.md

```diff
@@ -13,15 +13,30 @@ This document lists the most frequently asked questions about TiDB Cloud Starter
 
 ### What is TiDB Cloud Starter?
 
-TiDB Cloud Starter offers the TiDB database with full HTAP capabilities for you and your organization. It is a fully managed, auto-scaling deployment of TiDB that lets you start using your database immediately, develop and run your application without caring about the underlying nodes, and automatically scale based on your application's workload changes.
+TiDB Cloud Starter (formerly Serverless) offers the TiDB database with full HTAP capabilities for you and your organization. It is a fully managed, auto-scaling deployment of TiDB that lets you start using your database immediately, develop and run your application without caring about the underlying nodes, and automatically scale based on your application's workload changes.
+
+### What is the relationship between TiDB Cloud Starter and TiDB Cloud Serverless?
+
+TiDB Cloud Starter is the new name for TiDB Cloud Serverless, effective August 12, 2025.
+
+Before it was renamed to Starter, the Serverless tier of TiDB Cloud served as the entry point for thousands of developers, providing a production-ready database that scales automatically, starts in seconds, and costs nothing until you exceed a generous free quota.
+
+While "serverless" accurately describes how the service works behind the scenes, many first-time users told us this word felt abstract and overloaded with meanings.
+
+We renamed this entry tier to Starter so its purpose is crystal clear: it is the quickest way to begin building with TiDB Cloud. Everything you enjoy with Serverless remains unchanged:
```

Comment:
Suggest we use AI to shorten this paragraph so we can keep each FAQ question short and concise.

Reply:
Updated in https://github.com/pingcap/docs/pull/21437